src/components/reviews/reviewsList.jsx

```diff
@@ -38,8 +40,10 @@ class ReviewsList extends React.Component {
     let current = new Set(this.state.filters);
     if (current.has(stars)) {
       current.delete(stars);
+      event.target.setAttribute("style", "font-weight:normal");
```

Comment:
This is a real cool way of implementing style. A better approach than J-query I would think for this scenario.